script/build-artifact.sh

```diff
@@ -44,9 +44,15 @@ mkdir -p "${STAGING}" "${EXTRAS_STAGING}"
 
 echo "Creating \"${TARBALL}\" and \"${EXTRAS_TARBALL}\""
 
+# Use linux config file as default
+RELEASE_FOLDER="linux"
+if [ ${OS} == "windows" ]; then
+    RELEASE_FOLDER="windows"
+fi
+
 # Copy in the contents under release/
-cp -r release/spire/* "${STAGING}"
-cp -r release/spire-extras/* "${EXTRAS_STAGING}"
+cp -r release/spire-${RELEASE_FOLDER}/* "${STAGING}"
```

Comment:
I wonder if instead of:
```
release/spire-linux
release/spire-linux-extras
release/spire-windows
release/spire-windows-extras
```

We could instead have:
```
release/linux/spire
release/linux/spire-extras
release/windows/spire
release/windows/spire-extras
```